src/libs/DraftCommentUtils.ts

```diff
@@ -1,14 +1,14 @@
-import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {DraftReportComments} from '@src/types/onyx';
 
-let draftCommentCollection: OnyxCollection<string> = {};
+let draftCommentCollection: OnyxEntry<DraftReportComments> = {};
```

Comment:
NABish, but it's not really a collection key anymore

```suggestion
let draftComments: OnyxEntry<DraftReportComments> = {};
```